django_postgresql_reconnect/decorator.py

```diff
@@ -0,0 +1,21 @@
+#  Copyright © 2022 Ingram Micro Inc. All rights reserved.
+
+from functools import wraps
+
+from django_postgresql_reconnect.helpers import check_pgsql_connections
+
+
+def check_pgsql_connections_decorator(f):
+    """ Check PostgreSQL connection before running function
+
+        import django_postgresql_reconnect
+
+        @django_postgresql_reconnect.decorator
+        def some_func():
+            ...
+    """
+    @wraps(f)
+    def wrap(*args, **kwargs):
+        check_pgsql_connections()
```

Comment:
Shouldn't it be `try/except/finally` here?

Reply:
What for? If we can't reconnect, an exception will be raised. I think we shouldn't do anything with it.